release_notes/v2.2.0.md

```diff
@@ -4,23 +4,166 @@ v2.2.0 Release Notes - July XX, 2020
+
+**Note: Fixes included in v2.1.1 release notes are also included in v2.2.**
 
 Changes
 -------
 
-- **FAB-17917: Change default CouchDB maxRetriesOnStartup property from 12 to 10**
+**FAB-17786: upgrade_dbs peer command now drops state CouchDB databases**
+
+Previously the upgrade_dbs command did not automatically drop state CouchDB
+databases and therefore a separate step was required to drop CouchDB data
+when upgrading to v2.x. upgrade_dbs command now automatically drops
+state CouchDB databases. CouchDB state database will get rebuilt on the first
+peer start after the upgrade to v2.x. CouchDB database service
+must be available when running upgrade_dbs command. Similarly,
+rebuild-dbs also drops state CouchDB data now, so that state database
+can be rebuilt on the next peer start.
+
+**FAB-17869: Allow TLS CAs with overlapping issuers**
+
+The client root TLS CA certificate pool construction didn't allow different issuers
+with the same subject name to exist in the CA cert pool. Different issuers with
+the same subject name are now allowed.
+
+**Fabric CouchDB tests have been updated from CouchDB 2.3.1 to CouchDB 3.1.0**
+
+Support is added for CouchDB 3.1.0 as the recommended and tested version of CouchDB.
+If prior versions are utilized, a Warning will appear in peer log.
+Note that CouchDB 3.1.0 requires that an admin username and password be set,
+while this was optional in CouchDB v2.x. See the
+[Fabric CouchDB documentation](https://hyperledger-fabric.readthedocs.io/en/release-2.2/couchdb_as_state_database.html#couchdb-configuration)
+for configuration details.
+Also note that CouchDB 3.1.0 default max_document_size is reduced to 8MB. Set a higher value if needed in your environment.
+
+**FAB-17917: Peer CouchDB default `maxRetriesOnStartup` property has been updated**
+
+Peer property `peer.ledger.state.couchDBConfig.maxRetriesOnStartup` default has
+changed from 12 to 10. The time between retries doubles after each attempt.
+Therefore if CouchDB is not yet started, the peer start will now retry
+for about 2 minutes rather than 16 minutes before retries are exhausted.
+
+**FAB-16435: Peer gossip defaults have been updated**
 
-  The time between retries doubles after each attempt.
-  Therefore if CouchDB is not yet started, the peer start will now retry
-  for about 2 minutes rather than 16 minutes before retries are exhausted.
+Block dissemination via gossip may be removed in a future release,
+since it is more straightforward for peers to simply pull blocks from ordering service.
+The gossip defaults have been updated to prepare users for this possible change, so that peers by default
+pull blocks from ordering service, do not use leader election, and do not use block transfer across peers.
+Additionally, two block cache default sizes have been lowered to reduce the memory
+used by a peer when it has joined many channels.
+The new defaults are as follows:
```

Comment:
Are these default settings only for new peers that are created with Fabric 2.2?

Reply:
I'm adding this clarification:
Default configuration values are included in the peer docker image, therefore if you apply
the new peer image the new defaults will be effective unless you specifically override them
in your configuration.